baseband/vlbi_base/base.py

```diff
@@ -120,7 +120,10 @@ def _get_time(self, header):
 
     @lazyproperty
     def start_time(self):
-        """Start time of file."""
+        """Start time of the file.
+
+        See also `time` for the time of the sample pointer's current offset.
```

Comment:
Does the single ticks here work for the documentation? (I see lots of build failures so perhaps not...)

Reply:
Maybe also refer to `stop_time`...